WcaOnRails/app/views/team_members/_team_member_form.html.erb

```diff
@@ -0,0 +1,21 @@
+  <%= simple_form_for [@committee, @team, @team_member], html: { class: 'form-horizontal are-you-sure no-submit-on-enter' }, wrapper: :horizontal_form,
+    wrapper_mappings: {
+      check_boxes: :horizontal_radio_and_checkboxes,
+      radio_buttons: :horizontal_radio_and_checkboxes,
+      file: :horizontal_file_input,
+      boolean: :horizontal_boolean
+    }  do |f| %>
+    <%= render 'shared/error_messages', object: f.object %>
+    <% if @team_member.persisted? %>
+      <%= f.input :user_id, disabled: true, required: true, as: :user_ids, only_one: true %>
+      <%= f.association :committee_position, collection: @committee.committee_positions, label_method: :name, value_method: :id, disabled: true %>
+    <% else %>
+      <%= f.input :user_id, required: true, as: :user_ids, only_one: true %>
+      <%= f.association :committee_position, collection: @committee.committee_positions, label_method: :name, value_method: :id %>
+    <% end %>
```

Comment:
What about having two inputs here and using
`disabled: @team_member.persisted?`